lib/cli/base.js

```diff
@@ -13,6 +14,16 @@ export default function(cli) {
 		"This option will disable resolution of node package dependencies.",
 			type: "string"
 		})
+		.option("workspace-config", {
+			describe: "Path to workspace configuration file in YAML format",
```

Comment:
alias `wc`?

Reply:
I think we should stick to one-letter abbreviations. Yargs also makes a differentiation: One letter options only need one dash `-c`, two letters and more require two dashes `--wc` which might make it more confusing than helpful 😕

Reply:
good to know. fine for me. thx for clarification